src/ProjectTemplates/test/RazorComponentsTemplateTest.cs

```diff
@@ -73,6 +73,56 @@ public async Task RazorComponentsTemplateWorks()
             }
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        [Flaky("https://github.com/aspnet/AspNetCore-Internal/issues/2407", FlakyOn.AzP.Windows)]
```

Comment:
This issue is closed. Did you mean to link https://github.com/aspnet/AspNetCore-Internal/issues/2329?

Reply:
I was just copying the same link from another test. I guess multiple need to be updated.